backend/azure/keyvault.go

```diff
@@ -4,56 +4,61 @@
 // Copyright 2024-present Datadog, Inc.
 // Copyright (c) 2021, RapDev.IO
 
+// Package azure allows to fetch secrets from Azure keyvault service
 package azure
 
 import (
 	"context"
 	"encoding/json"
 	"errors"
+
 	"github.com/Azure/azure-sdk-for-go/services/keyvault/v7.1/keyvault"
 	"github.com/DataDog/datadog-secret-backend/secret"
 	"github.com/mitchellh/mapstructure"
 	log "github.com/sirupsen/logrus"
 )
 
-type AzureKeyVaultBackendConfig struct {
-	AzureSession AzureSessionBackendConfig `mapstructure:"azure_session"`
-	BackendType  string                    `mapstructure:"backend_type"`
-	ForceString  bool                      `mapstructure:"force_string"`
-	KeyVaultURL  string                    `mapstructure:"keyvaulturl"`
-	SecretId     string                    `mapstructure:"secret_id"`
+// KeyVaultBackendConfig contains the configuration to connect for Azure backend
+type KeyVaultBackendConfig struct {
+	Session     SessionBackendConfig `mapstructure:"azure_session"`
+	BackendType string               `mapstructure:"backend_type"`
+	ForceString bool                 `mapstructure:"force_string"`
+	KeyVaultURL string               `mapstructure:"keyvaulturl"`
```

Comment:
```suggestion
	KeyVaultURL string               `mapstructure:"key_vault_url"`
```
I don't know if this one is relevant